agent/ecr/model/generate.go

```diff
@@ -13,4 +13,4 @@
 
 package model
 
-//go:generate go run ../../gogenerate/awssdk.go -typesOnly=false
```

Comment:
Got it. In the spirit of "self documenting code", can you either modify the commit message or add a comment to state the same?